packages/share/android/src/main/java/io/flutter/plugins/share/Share.java

```diff
@@ -47,4 +61,150 @@ void share(String text, String subject) {
       activity.startActivity(chooserIntent);
     }
   }
+
+  void shareFiles(List<String> paths, List<String> mimeTypes, String text, String subject)
+      throws IOException {
+    if (paths == null || paths.isEmpty()) {
+      throw new IllegalArgumentException("Non-empty path expected");
+    }
+
+    clearExternalShareFolder();
+    ArrayList<Uri> fileUris = getUrisForPaths(paths);
+
+    Intent shareIntent = new Intent();
+    if (fileUris.size() == 1) {
```

Comment:
Should never be 0 because the dart side handles it. But just to be sure I now added an `isEmpty` check that then just does a normal share with `share(text, subject)`.